expyfun/_input_controllers.py

```diff
@@ -84,6 +99,17 @@ def _on_pyglet_keypress(self, symbol, modifiers, emulated=False):
             this_key = this_key.lstrip('_').lstrip('NUM_')
         self._keyboard_buffer.append((this_key, key_time))
 
+    def _on_pyglet_keyrelease(self, symbol, modifiers, emulated=False):
```

Comment:
My mistake. This will work (for pyglet windows). We'll probably need to figure out how to get it working with TDT before merging it into master though. And possibly a refactor to make it more DRY.